catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -335,15 +351,13 @@ public void configure() throws Exception {
                 RouteDefinition routeDefinition = from(inbox);
 
                 if (attributeOverrides != null) {
-                    String attributeOverrideString = attributeOverrides.stream()
-                            .map(String::trim)
-                            .collect(Collectors.joining(","));
-                    routeDefinition.setHeader(Constants.ATTRIBUTE_OVERRIDES_KEY, simple(
-                            attributeOverrideString));
+                    routeDefinition.setHeader(Constants.ATTRIBUTE_OVERRIDES_KEY)
+                            .constant(attributeOverrides);
```

Comment:
This is my ignorance of Camel showing: why is this being set as `.constant()` on the header instead of calling `setHeader` with two params where the second is the output of a call to `.simple()`? That's how it appears it's being done everywhere else.